src/queries.js

```diff
@@ -129,11 +165,40 @@ function queryByAltText(container, alt) {
 function getByAltText(container, alt) {
   const el = queryByAltText(container, alt)
   if (!el) {
-    throw new Error(`Unable to find an element with the alt text: ${alt}`)
+    const htmlElement = checkDebugLimit(
+      prettyFormat(container, {
+        plugins: [DOMElement],
+        printFunctionName: false,
+        highlight: true,
+      }),
+    )
+    throw new Error(
+      `Unable to find an element with the alt text: ${alt} \nHere is the state of your container: \n${htmlElement}`,
+    )
   }
   return el
 }
 
+function checkDebugLimit(debugContent) {
```

Comment:
The name is confusong: it shouldn't be "check" (would return boolean), it is more of a "trimToDebugLimit".